python/GafferUI/SwitchUI.py

```diff
@@ -58,9 +58,23 @@
 				of 0 chooses the first input, 1 the second and so on. Values
 				larger than the number of available inputs wrap back around to
 				the beginning.
+				""",
+
+				"nodule:type", "",
+
+			],
+
+			"in" : [
+
+				"description",
 				"""
+				The array of inputs to choose from. One of these is chosen
+				by the index plug to be passed through to the output.
+				""",
+
+				"nodule:type", "GafferUI::CompoundNodule",
```

Comment:
I was going to suggest that originally, but then I wasn't sure if having the widget for an out color was useful?